pages/content-api/filters.mdx

```diff
@@ -26,7 +26,7 @@ The syntax of the `where` clause is shown in the example given below.
   </Tabs.Tab>
   <Tabs.Tab>
 ```tsx
-// TypeScript code is coming soon...
+await unbody.get.ClassName.where(...conditions).select(...fields).exec()
```

Comment:
Not clear if these are placeholders. we can do things like .get.<classname>.where(...)